core/src/com/mygdx/game/core/util/DirtyInt.java

```diff
@@ -0,0 +1,9 @@
+package com.mygdx.game.core.util;
+
+import lombok.Data;
+
+@Data
+public class DirtyInt {
```

Comment:
Co to jest

Reply:
https://stackoverflow.com/a/1745899/3517028

To takie coś co pozwala mi na wprowadzenie trochę cachowania i leniwego obliczania

Reply:
Ale używasz tego gdziekolwiek?

Reply:
aaaa, faktycznie, bo usunąłem te zmiany, które ostatecznie mi nie działały xD

dobrze, że to zauważyłeś